scripts/xml_to_csv/xml_to_csv.py

```diff
@@ -0,0 +1,40 @@
+# Written By: Jodi Gunawan
+# Date: Nov 15th, 2018
+# Description:  This script creates csv file from xml input file under the file train_labels
+
+import os
+import glob
+import pandas as pd
+import xml.etree.ElementTree as ET
+
+
```

Comment:
Could put a quick function definition comment here, I know the function name is pretty explanatory but it's a good habit to get into.